integration-tests/http/src/main/java/org/apache/camel/quarkus/component/http/it/HttpResource.java

```diff
@@ -155,8 +155,9 @@ public String nettyHttpGet(@QueryParam("test-port") int port) {
     @GET
     @Produces(MediaType.TEXT_PLAIN)
     public String nettyHttpGetHttps() {
+        //        String truststore = System.getProperty("javax.net.ssl.trustStore");
```

Comment:
Lets remove if not needed.

Reply:
I'm sorry, I've missed that, I'll fix it